### PR TITLE
Kumaran changes

### PR DESCRIPTION
**Pull request by KChan440**
These are good changes. A++++

### CODE REVIEW

**Comment by max-su** [comment]
so bad

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 Android Repository!!
+Kumaran change
```
Comment (ding-harrison) [inline README.md:2]: Trash wtf

Comment (KChan440) [inline README.md:2]: sike u suck m8

**Comment by Ceielle** [comment]
*insert poop emoji here*

File: README.md (hunk 1 above)
Comment (alu416) [inline README.md:2]: ._.

Comment (ding-harrison) [inline README.md:2]: dont suck as hard as you bruh

Comment (Ceielle) [inline README.md:2]: _insert poop emoji here_

**Pull request merged**